src/OrchardCore.Modules/OrchardCore.Users/TimeZone/Services/UserTimeZoneService.cs

```diff
@@ -9,10 +9,14 @@
 
 namespace OrchardCore.Users.TimeZone.Services;
 
-public class UserTimeZoneService
+/// <summary>
+/// Represents a time zone service for currently logged-in user.
+/// </summary>
+public class UserTimeZoneService : IUserTimeZoneService
 {
-    private const string EmptyTimeZone = "empty";
     private const string CacheKey = "UserTimeZone/";
+    private const string EmptyTimeZone = "empty";
```

Comment:
`_emptyTimeZone`

Reply:
It' a constant

Reply:
Yes private constant.

Reply:
Let's keep it consistent with `CacheKey`, shouldn't we?

Reply:
You can check Microsoft naming guidelines